Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gemspec
 
 gem "rake"
+gem "rails"
```

Comment:
Is this an oversight? We shouldn't need to bring in the entire Rails gem into jbuilder, right?